### PR TITLE
Implement LIC 3

### PR DESCRIPTION
**Pull request by teljoh**
This PR implements LIC 3 and test cases for it.

Fixes: #16

### CODE REVIEW

File: src/main/java/CMV.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import javax.swing.*;
```
Comment (elmaxe) [inline src/main/java/CMV.java:1]: I'm unsure why swing is imported here. Remove this line

**Review by elmaxe — APPROVED** [review]

File: src/main/java/CMV.java (hunk 1 above)
Comment (K2017) [reply src/main/java/CMV.java:1]: Agreed, might unknowingly pull in a dependency

**Review by K2017 — CHANGES_REQUESTED** [review]
See comments

**Review by andreaskth — APPROVED** [review]
Apart from importing java swing it looks all good to me, nice job 😃

**Review by K2017 — APPROVED** [review]
Removed swing import

**Pull request merged**